MarathonRecomp/api/Physics/Shape.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <Marathon.inl>
+
+namespace Phys
```

Comment:
If this is for Havok classes, rename this namespace to `hk330_b2`. Otherwise, rename to `Physics` or find a more appropriate namespace.